src/components/views/settings/tabs/SecuritySettingsTab.js

```diff
@@ -0,0 +1,242 @@
+            .map((u) => <IgnoredUser userId={u} onUnignored={this._onUserUnignored} key={u} />);
+
+        return (
+            <div className='mx_SettingsTab_section'>
+                <span className='mx_SettingsTab_subheading'>{_t('Ignored users')}</span>
+                <div className='mx_SettingsTab_subsectionText'>
+                    {userIds}
+                </div>
+            </div>
+        );
+    }
+
+    _renderRejectInvites() {
+        const invitedRooms = MatrixClientPeg.get().getRooms().filter((r) => {
+            return r.hasMembershipState(MatrixClientPeg.get().getUserId(), "invite");
+        });
+        if (invitedRooms.length === 0) {
+            return null;
+        }
+
+        const onClick = this._onRejectAllInvitesClicked.bind(this, invitedRooms);
+        return (
+            <div className='mx_SettingsTab_section'>
+                <span className='mx_SettingsTab_subheading'>{_t('Bulk options')}</span>
+                <AccessibleButton onClick={onClick} kind='danger' disabled={this.state.rejectingInvites}>
+                    {_t("Reject all %(invitedRooms)s invites", {invitedRooms: invitedRooms.length})}
+                </AccessibleButton>
+            </div>
+        );
+    }
+
+    render() {
+        const DevicesPanel = sdk.getComponent('views.settings.DevicesPanel');
+        const SettingsFlag = sdk.getComponent('views.elements.SettingsFlag');
+
+        let keyBackup = null;
+        if (SettingsStore.isFeatureEnabled("feature_keybackup")) {
+            const KeyBackupPanel = sdk.getComponent('views.settings.KeyBackupPanel');
+            keyBackup = (
+                <div className='mx_SettingsTab_section'>
+                    <span className="mx_SettingsTab_subheading">{_t("Key backup")}</span>
+                    <div className='mx_SettingsTab_subsectionText'>
+                        <KeyBackupPanel />
+                    </div>
+                </div>
+            );
+        }
+
+        return (
+            <div className="mx_SettingsTab mx_SecuritySettingsTab">
+                <div className="mx_SettingsTab_heading">{_t("Security & Privacy")}</div>
+                <div className="mx_SettingsTab_section">
+                    <span className="mx_SettingsTab_subheading">{_t("Devices")}</span>
+                    <div className='mx_SettingsTab_subsectionText'>
+                        <DevicesPanel />
+                    </div>
+                </div>
+                {keyBackup}
+                {this._renderCurrentDeviceInfo()}
+                <div className='mx_SettingsTab_section'>
+                    <span className="mx_SettingsTab_subheading">{_t("Analytics")}</span>
+                    <div className='mx_SettingsTab_subsectionText'>
+                        {_t('Riot collects anonymous analytics to allow us to improve the application.')}
+                        &nbsp;
+                        {_t('Privacy is important to us, so we don\'t collect any personal or ' +
```

Comment:
btw I would generally use double quotes for longer, English strings like this that are likely to contain apostrophes to avoid the escaping.